docusaurus/docs/Android/04-compose/03-channel-components/01-channels-screen.mdx

```diff
@@ -1,6 +1,6 @@
 # ChannelsScreen
 
-The easiest way to set up a screen that shows the active user's channels and give them the ability to search for a specific channel is to use the `ChannelsScreen`.
+The easiest way to set up a screen that shows the active user's channels and gives them the ability to search for a specific channel is to use the `ChannelsScreen`.
 
```

Comment:
Not sure what does it mean to `remove the start list of components used (it's not scalable)`?